roles/keystone-setup/tasks/main.yml

```diff
@@ -42,6 +42,7 @@
                  tenant={{ item.tenant }}
                  token={{ admin_token }}
   with_items: keystone.users
+  when: "{{ item.name }}.enabled is undefined or {{item.name}}.enabled|bool"
```

Comment:
Yeah, it may be that way in other places, but I'd prefer to clean it up. In fact, the final bool filter may not even be necessary, just needs some trial and error to see if it can work.